cargo-dist/src/backend/ci/github.rs

```diff
@@ -74,6 +75,8 @@ pub struct GithubCiInfo {
     pub tag_namespace: Option<String>,
     /// `gh` command to run to create the release
     pub release_command: String,
+    /// Which phase to create the release at
+    pub release_phase: GithubReleasePhase,
```

Comment:
I would prefer that this be called github_release_phase, as this does not e.g. affect axo releases.

Reply:
Actually I see we've broke this on several fields already so nvm, rather not add more churn.